repositories/cyclonedds.BUILD.bazel

```diff
@@ -181,6 +181,7 @@ cmake(
     ),
     copts = [
         "-w",
+        "-D__FILE__=\"redacted\"",
```

Comment:
Is this determinism related? Doesn't Bazel already set this via compiler flags?

Reply:
Generally yes, but CMake involvement caused issue here that were solved by this

Reply:
Just for my understanding, what was the issue?

Reply:
absolute paths in binaries, which causes indeterministic builds